debugging-with-ssh-access.md

```diff
@@ -0,0 +1,81 @@
+Often the best way to troubleshoot failures in your pipelines is to SSH into a
```

Comment:
I will add a **TOC** once the page is ready. Also, we will need an `Overview` section at the beginning of the page.

Reply:
Good point. I opted to not follow this rule in this guide for two reason:

- The first information on the page should give a high level explanation of a concept. Jumping immediately into a TOC delays that information and makes it harder to understand the key points.

Example kubernetes docs: https://kubernetes.io/docs/concepts/configuration/secret/

- The overview section can be very useful for longer documents. I feel that for short guides like this, it is an unnecessary duplication (having both the intro at the begginging of the doc + an overview).